mirar/processors/database/database_inserter.py

```diff
@@ -133,13 +133,16 @@ def _apply_to_images(self, batch: ImageBatch) -> ImageBatch:
                 logger.error(err)
                 raise ImageBatchDatabaseExporterError(err) from exc
 
-            if len(np.unique(values)) > 1:
-                err = (
-                    f"Key {column} differs across images in the batch, cannot export"
-                    f"it to database."
-                )
-                logger.error(err)
-                raise ImageBatchDatabaseExporterError(err)
+            logger.debug(f"{column}, {values}")
+            check_nones = [x is None for x in values]
+            if not np.all(check_nones):
```

Comment:
Please remove this! I just replaced it to pd.unique in my PR, that's a smaller fix.

Reply:
removed